app/experimenter/static/rapid/components/forms/TabRoutes.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import {
+  NavLink,
+  Route,
+  RouteProps,
+  Switch,
+  useRouteMatch,
+} from "react-router-dom";
+
+/**
+ * Helper function to join paths without duplicating slashes.
+ * Adds a trailing slash to be inline with Experimenter's style of URL
+ */
+export function pathJoinWithTrailingSlash(...paths: Array<string>): string {
+  return `${paths.join("/")}/`.replace(/\/{1,}/g, "/");
+}
+
```

Comment:
It's too bad the browser doesn't expose a url joiner, I only find this:

https://www.npmjs.com/package/url-join

On the one hand adding a new dependency isn't great, but on the other there are a lot of nuances in URL paths that we might accidentally miss, so maybe adding a dependency isn't the worst?

Reply:
I think I'd rather leave it as-is for now unless we run into issues / need to expand the scope, and which point replacing this function (or the whole thing) might be worth doing